lib/matplotlib/rcsetup.py

```diff
@@ -175,6 +175,20 @@ def validate_string_or_None(s):
         raise ValueError('Could not convert "%s" to string' % s)
 
 
+def validate_stringlist_or_string(s):
+    """convert s to string or raise"""
+    if s is None or s == 'None':
+        return ""
+    try:
+        if isinstance(s, str):
+            return s
+        if isinstance(s, list):
```

Comment:
This will cause tuples to fail which seems less than great, should this just be an `else` clause?

Reply:
Probably any iterable that's not a string.

Reply:
So, that should be "isinstance(s, Iterable)"?

Reply:
See 5b6e000.  I hope this is correct.